src/banana/data/basis_rotation.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
```

Comment:
This file is actually duplicated from `eko`, so we should rely on one of the two. Since actually `banana` is only a dependency of `ekomark`, and `ekomark` is of course not a dependency of `eko` (not even the opposite at the moment, but this is irrelevant), then I would make `banana` to depend on `eko`.

Reply:
> not even the opposite at the moment, but this is irrelevant

maybe not explicit - but implicit for sure, since there is a `eko.run_dglap` call somewhere ...

Reply:
Yes, I know, that's why I specified it's irrelevant: they would end up both depending on the same, thus `ekomark` will depend directly and indirectly on `eko`.